nservicebus/upgrades/5to6/index.md

```diff
@@ -10,20 +10,189 @@ related:
+
+**Upgrade one endpoint at a time.**
+
+There is one common issue with upgrading a single endpoint at a time. If the endpoints in a solution share a common libary then upgrading one endpoint might lead to changes in the common library and that necessitates changes in all of the other endpoints that rely on the common library at the same time. The recommended approach to dealing with this is to create a copy of the common libary for the new endpoint and to upgrade it along with the endpoint. When the time comes to upgrade the second endpoint, change it's dependency to point to the new, upgraded, version of the common library. When using this approach, other changes to the common library should be minimized as they will need to be reflected in both codebases.
+
+The process of upgrading each endpoint is going to follow a common sequence of steps. Being able to repeatably apply those steps is key to the success of the upgrade project. The recommended approach is to upgrade a simple and low risk endpoint first to ensure that the process is well understood before tackling the endpoints that make up the core of the solution. Endpoints that send email or generate documents are often good candidates for this. When selecting the first endpoint to upgrade look for a small number of reasonably straightforward handlers and a small amount of NServiceBus configuration. It is worth considering selecting a simple endpoint to upgrade even if it will not take advantage of Version 6 features to practice the upgrade process. 
+
+
+### Move to .NET 4.5.2
 
 The minimum .NET version for NServiceBus Version 6 is .NET 4.5.2.
 
-**Users must update all projects (that reference NServiceBus) to .NET 4.5.2 before updating to NServiceBus Version 6.**
+**All projects (that reference NServiceBus) must be updated to .NET 4.5.2 before updating to NServiceBus Version 6.**
 
 It is recommended to update to .NET 4.5.2 and perform a full migration to production **before** updating to NServiceBus Version 6.
 
 For larger solutions the Visual Studio extension [Target Framework Migrator](https://visualstudiogallery.msdn.microsoft.com/47bded90-80d8-42af-bc35-4736fdd8cd13) can reduce the manual effort required in performing an upgrade.
 
 
-## IBus, IStartableBus and the Bus Static class are now obsolete
+### Update NServiceBus dependencies
+
+All NServiceBus dependencies for an endpoint project are managed via NuGet. Open the Manage NuGet Packages window for the endpoint project, switch to the Updates tab and look for packages that start with NServiceBus. Update each one to the latest Version 6 package.
+
+(screenshot?)
+
+NOTE: All of the NuGet packages are currently available as prerelease builds so the Include prerelease option must be selected by either checking the box in the Manage NuGet Packages window or by including the `-Pre` flag in the Package Manager Console. 
```

Comment:
can u add an issue to come back and clean this up. add it to a "v6 after rtm" milestone